ceph.spec.in

```diff
@@ -150,6 +151,12 @@ BuildRequires:	libbz2-devel
 %if 0%{with tcmalloc}
 BuildRequires:	gperftools-devel
 %endif
+%if 0%{with xio}
+BuildRequires:  librdmacm-devel
+BuildRequires:  libibverbs-devel
+BuildRequires:  libnuma-devel
+BuildRequires:  libevent-devel
```

Comment:
actually those are Accelio requirements. so the requirements should actually be in libxio.spec (accelio package name).
ceph requirement should be libxio-devel for build and libxio for install.